docs/howto/manage-clouds.md

```diff
@@ -11,28 +11,26 @@ This document shows how to manage your existing cloud(s) with Juju.
 ## Add a cloud
 
 ```{ibnote}
-See also: {ref}`list-of-supported-clouds` > `<cloud name>`
+See also: {ref}`list-of-supported-clouds`
 ```
 
-```{important}
-This step is typically required only for non-localhost private clouds.
-```
+The procedure for how to add a cloud definition to Juju depends on whether the cloud is a machine (traditional, non-Kubernetes) cloud or rather a Kubernetes cloud -- more below.
```

Comment:
s/or rather a/or a/

Reply:
Fixed (here and in 2 other places). Thanks!